docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -14,11 +14,11 @@ The cluster can be configured via a YAML file. This custom resource specifies th
     metadata:
       name: simple-kafka
     spec:
-      version: 2.8.1
+      version: 3.1.0
       zookeeperConfigMapName: simple-kafka-znode
       opa:
         configMapName: simple-opa
-        authorizerClassName: com.bisnode.kafka.authorization.OpaAuthorizer
+        authorizerClassName: org.openpolicyagent.kafka.OpaAuthorizer
```

Comment:
I think we can leave out `authorizerClassName` (which will be included by the product-config). This would make the examples work for both 2.8.1 and 3.1.0?

Reply:
Looks like it's currently required if `opa` is non-null.

Reply:
Yeah https://github.com/stackabletech/kafka-operator/blob/9b973fa9a35dcfdd66b5a41b82416229cba86d77/rust/crd/src/lib.rs#L113
I think we can just remove that? No point in having that configurable by the user?

Reply:
https://github.com/stackabletech/kafka-operator/issues/350